src/tests/ftest/util/test_utils_container.py

```diff
@@ -368,6 +369,103 @@ def create(self, uuid=None, con_in=None, acl_file=None):
         self.uuid = self.container.get_uuid_str()
         self.log.info("  Container created with uuid %s", self.uuid)
 
+    @fail_on(DaosApiError)
+    def create_snap(self, pool=None, cont=None, snap_name=None, epoch=None,
```

Comment:
I don't think we should provide pool, cont, svc, and sys_name as parameters. After TestContainer object is created, we expect the caller to call create, which sets the container UUID to its member. We also expect the pool to exist before creating a TestContainer object, so pool, svc, and sys_name of its pool member should be used here. So I think line 386 to 393 should be removed and the values in kwargs should be adjusted to use the members.

Reply:
Done